src/Uno.UI.RuntimeTests/MUX/Utilities/IdleSynchronizer.cs

```diff
@@ -8,6 +8,7 @@ namespace MUXControlsTestApp.Utilities
 {
 	internal class IdleSynchronizer
 	{
+		[Obsolete("Don't use IdleSynchronizer.Wait. Use 'await TestServices.WindowHelper.WaitForIdle()' instead.")]
```

Comment:
Maybe worth even banning the API?

Reply:
As it's internal, obsoletion should be okay.

Usually, the BannedApiAnalyzer is useful for two scenarios:

1. An API that comes from a package/library that we don't control.
2. An API that we control but is either public and we don't want to obsolete it publicly, or we want the obsoletion for specific project.

But if it's an API that we want to "ban everywhere", I think Obsolete should be good.